commit a288b38a89da8102db08ec11118ded815654de42 (refs/changes/97/497/1)
Author: Ryan Zezeski <rpz@joyent.com>
Date:   2016-09-16T09:52:48-04:00 (3 years, 1 month ago)
    
    OS-5330 zoneadm mounting an lx or joyent branded zone fails

diff --git a/overlay/generic/etc/zones/SUNWdefault.xml b/overlay/generic/etc/zones/SUNWdefault.xml
index cfe27a0f..070412f6 100644
--- a/overlay/generic/etc/zones/SUNWdefault.xml
+++ b/overlay/generic/etc/zones/SUNWdefault.xml
@@ -28,5 +28,5 @@
 
 <!DOCTYPE zone PUBLIC "-//Sun Microsystems Inc//DTD Zones//EN" "file:///usr/share/lib/xml/dtd/zonecfg.dtd.1">
 
-<zone name="default" zonepath="" autoboot="false" brand="liveimg">
+<zone name="default" zonepath="" autoboot="false" brand="joyent">
 </zone>
